js/dom.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable func-names */
+// eslint-disable-next-line func-names
+// eslint-disable-next-line no-unused-vars
+const dom = (function () {
+  function getElement(id) {
```

Comment:
[Optional]
It would be great to practice `ES6` by using arrow function.

Reply:
Thank you for the suggestion we've used it now